src/components/header.js

```diff
@@ -23,11 +23,20 @@ const Header = ({ siteTitle }) => (
             textDecoration: 'none',
           }}
         >
-          {siteTitle}
+          {siteData.title}
         </Link>
       </h1>
+      <h5
+        style={{
+          color: 'white',
+          textDecoration: 'none',
+          margin: '16px 0 0 0',
+        }}
+      >
+        {siteData.subTitle}
+      </h5>
```

Comment:
h5 after h1?

headers should not be used just for different font sizes

Reply:
Oh maybe I should do a bit more reading. But the gist of what happening here is you're saying headers should be used only for titles in document type views and not for just visual purposes?